### DISCUSSION TITLE
heat pump performance simulation

### DISCUSSION THREAD

**Discussion by giuseppe072000** [oemof/tespy#328] [Ideas]
Hi,

I`m looking for a script to run a  consumption simulation ( KWh)  for a ASHP using  :
-data sheet of the  HP
-temperature and Humidity/HR  for a specific location  in winter(  288 readings/24/h  )

What software/script is possible to make this simulation ?

Thanks

 

https://keymark.eu/en/products/heatpumps/certified-products
 

https://www.wunderground.com/history/daily/pl/warsaw/EPWA/date/2022-2-11   

EN 14825

Low temperature

Medium temperature

ηs

205 %

137 %

Prated

9.17 kW

7.67 kW

SCOP

5.19

3.49

Tbiv

-7 °C

-7 °C

TOL

-10 °C

-10 °C

Pdh Tj = -7°C

8.11 kW

6.78 kW

COP Tj = -7°C

3.23

2.24

Cdh Tj = -7 °C

0.90

0.90

**Comment by fwitte** [comment]
Hi Giuseppe,

thank you for reaching out. I do have some trouble following your plans here. What will be your input and what outputs do you expect? I understand, you have some ambient temperature readings and want to predict power consumption, is that correct?

For the input: What is the target temperature, what is the working fluid, what are the component’s specifics like temperature differences and efficiencies, how much heat should be provided? 

Best regards 

Francesco
